packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1220,6 +1220,41 @@ export default class NewVm extends BaseComponent {
                 </Item>
               )}
             </LineItem>
+            <br />
```

Comment:
I think we should be able to factorize this part with the component below

Reply:
What if we create a separate function for the ISO option and call it where it's needed instead of repeating this part of the code ?

Reply:
You can probably simply put the selector outside of the ternary operator:
```jsx
{this._isDiskTemplate ? (
  // SSH, cloud config, ...
) : (
  // PXE, ...
)}
// ISO selector
```

Reply:
What do you mean reversed? The ISO selector should always be at the bottom, right?

Reply:
I don't know if it should but before first modification, it was at the top :
![Screenshot from 2023-11-16 15-38-30](https://github.com/vatesfr/xen-orchestra/assets/119158464/77c6e8b6-3cba-4589-b351-7e8a3eccff51)

Reply:
Ah I get it now, it's reversed compared to before those changes. So yes, it's fine.